### ISSUE TITLE
There's no need to lint the code inside textarea

### ISSUE THREAD

**Issue by elecsiuz** [htmllint/htmllint#194]
Everything between <textarea></textarea> is rendered as plain text.
Please consider a textarea-ignore option, Thanks!

**Comment by mlochbaum** [comment]
Good idea. There are a handful of elements that need special handling (svg is totally broken; images should allow height and width attributes by default). Figuring out a framework for these is probably the next big project for htmllint after making it possible to work with or ignore html extensions. Although htmllint development happens on a scale of months at the best of times so that may not mean much.

**Comment by emmercm** [comment]
Similarly, content in `<code></code>` should be ignored as well, such as `<code><img src></code>` which fails `img-req-alt`.
